cmd/localstack/main.go

```diff
@@ -188,6 +190,12 @@ func main() {
 		SetLogsEgressAPI(localStackLogsEgressApi).
 		SetTracer(tracer)
 
+	// Corresponds to the 'AWS_LAMBDA_RUNTIME_API' environment variable.
+	// We need to ensure the runtime server is up before the INIT phase,
+	// but this envar is only set after the InitHandler is called.
+	runtimeAPIAddress := "127.0.0.1:9001"
+	sandbox.SetRuntimeAPIAddress(runtimeAPIAddress)
```

Comment:
I presume this is just to make sure the ping address and the runtime API one is the same, right? Nothing functional changes here?